server/integration_tests/test_data/detection_api_multivar/compareobesityvs.poverty/debug_info.json

```diff
@@ -5,104 +5,208 @@
+                "Count_Person_Urban_BelowPovertyLevelInThePast12Months",
+                "dc/topic/sdg_1.2.1",
+                "Count_Household_FamilyHousehold_With2Worker_BelowPovertyLevelInThePast12Months",
+                "Count_Person_Male_BelowPovertyLevelInThePast12Months",
+                "Count_Household_FamilyHousehold_With3OrMoreWorker_BelowPovertyLevelInThePast12Months",
+                "dc/topic/Malnutrition"
               ]
             }
           ]
         }
       ]
     },
-    "Query": "compare obesity vs poverty",
+    "Query": "compare obesity vs. poverty",
     "SV": [
-      "dc/topic/Poverty",
-      "dc/topic/PovertyByRace",
-      "dc/topic/SDG_1",
       "Percent_Person_Obesity",
-      "dc/topic/sdg_1.2",
-      "dc/topic/PovertyByGender",
-      "dc/topic/PovertyByLanguageSpokenAtHome",
-      "dc/topic/sdg_1.2.2",
-      "Count_Household_HouseholderAge65OrMoreYears_FamilyHousehold_BelowPovertyLevelInThePast12Months",
       "WHO/NCD_BMI_30A",
+      "dc/4lvmzr1h1ylk1",
+      "dc/x13tvt8jsgrm4",
+      "dc/topic/Health",
+      "dc/topic/HealthBehavior",
+      "dc/topic/Diabetes",
+      "dc/topic/HealthConditions",
+      "dc/topic/PhysicalActivity",
+      "WHO/NUTOVERWEIGHTPREV",
+      "Count_Person_Upto4Years_Overweight_AsFractionOf_Count_Person_Upto4Years",
+      "dc/topic/WHONonCommunicableDiseases",
+      "dc/topic/PopulationNormalWeight",
       "dc/topic/Malnutrition",
+      "Count_Person_Upto4Years_Female_Overweight_AsFractionOf_Count_Person_Upto4Years_Female",
+      "dc/topic/PreventativeHealthAndBehavior",
+      "Count_Person_Upto4Years_Male_Overweight_AsFractionOf_Count_Person_Upto4Years_Male",
+      "dc/topic/WHOHealthBehavior",
+      "dc/topic/HeartDisease",
+      "WHO/NCD_BMI_18A",
+      "Mean_PrePregnancyBMI_BirthEvent_LiveBirth",
+      "dc/topic/Cancer",
+      "Count_Death_EndocrineNutritionalMetabolicDiseases",
+      "dc/topic/PreventativeHealth",
+      "dc/topic/WHOChildrenNonCommunicableDiseases",
+      "dc/topic/BloodPressure",
+      "dc/topic/GlobalHealth",
       "dc/topic/sdg_2.1.1",
-      "sdg/SN_ITK_DEFC",
-      "Count_Household_WithFoodStampsInThePast12Months_BelowPovertyLevelInThePast12Months",
-      "dc/topic/PovertyByResidency",
-      "dc/topic/sdg_1",
-      "Count_Household_FamilyHousehold_BelowPovertyLevelInThePast12Months",
-      "dc/x13tvt8jsgrm4",
-      "dc/topic/FoodInsecurity",
-      "dc/topic/sdg_2.1.2",
-      "Count_Household_WithoutFoodStampsInThePast12Months_AbovePovertyLevelInThePast12Months",
-      "dc/4lvmzr1h1ylk1",
-      "Count_Household_MarriedCoupleFamilyHousehold_BelowPovertyLevelInThePast12Months",
-      "Count_Household_WithFoodStampsInThePast12Months_AbovePovertyLevelInThePast12Months",
-      "dc/topic/HealthEquity",
-      "dc/topic/sdg_1.2.1",
-      "Count_Person_BelowPovertyLevelInThePast12Months_TwoOrMoreRaces",
-      "Count_Person_Male_AbovePovertyLevelInThePast12Months_TwoOrMoreRaces",
-      "sdg/SI_POV_DAY1"
```

Comment:
Tests seems to have missed poverty SVs likely due to query change from 'vs' to 'vs.'

staging.datacommons.org/ does show these variables: https://screenshot.googleplex.com/6jM24c6njnXUv4z